src/casp/services/api/routers/cell_operations_router.py

```diff
@@ -103,6 +103,15 @@ async def annotate_cell_type_ontology_aware_strategy(
     )
 
 
+@cell_operations_router.get("/cache-info", response_model=schemas.CacheInfo)
+def get_cache_info(user: models.User = Depends(dependencies.authenticate_user)):
```

Comment:
Why do we need this method exposed? I thought it was intended to be removed before merging.

Reply:
@nmalfroy proposed we could keep it as admin-only.  I do think it could be useful for diagnosing issues, so I cleaned it up and added that admin-only check to it.  Thoughts?

Reply:
Yeah, I like the idea of keeping it as an API only (e.g. not client) endpoint for admins to verify the cache